microcosm_pubsub/dispatcher.py

```diff
@@ -74,6 +65,18 @@ def handle_message(self, message, bound_handlers):
             )
             return result
 
+    def find_handler(self, message, bound_handlers):
+        media_type = message.media_type
+        content = message.content
+
+        if content is None:
```

Comment:
The two error cases in this handler imply that we should not "count" the message at all (e.g. for metrics).

The second  check, at least, is a very common case because most daemons see a lot of message they do not process. It should not be treated the same as `SkipMessage`